quickstarts/WFLY-15746_WildFly_Application_Development_Guide.adoc

```diff
@@ -0,0 +1,83 @@
+= WildFly Application Development Guide
+:author:            Eduardo Martins
+:email:             emartins@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This document specifies the development of the Development Guide feature enhancement.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-15746[WFLY-15746]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/EAP7-1845[EAP7-1845]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:
+
+=== Affected Projects or Components
+
+WildFly (WFLY Documentation and Quickstarts Components)
+
+=== Other Interested Projects
+
+== Requirements
+
+=== Hard Requirements
+
+* The guide should have 2 major sections, one for first time/beginners app developers, and one for advanced app developers.
+
+* The major section for first time app developers should include the following new content:
+** How to develop a first basic application (helloworld quickstart) from scratch using Maven, a complete user story walkthrough, which should include:
+*** How to create the application Maven Project, and add Red Hat Maven repositories and BOMs dependencies
+*** How to package, deploy and run the application (war/bootable jar, web console)
+*** How to add logging to the application (basic usage)
+*** How to test the application
```

Comment:
Would it be worth in this section having something like "How to make the first modification" - i.e. the user has moved just beyond deploying an example we provide and made it their own with a small change?

Reply:
Please note that there is a quick start guide also, and that should provide just a deploy and run, in this one there is the coding of the hello world, and any further modification is then related with a specific technology (line 53)